doc/source/whatsnew/v3.0.0.rst

```diff
@@ -96,6 +96,13 @@ Deprecations
 - Deprecated :meth:`Timestamp.utcnow`, use ``Timestamp.now("UTC")`` instead (:issue:`56680`)
 -
 
+.. ---------------------------------------------------------------------------
+.. _whatsnew_300.prior_deprecations:
+
+Removal of prior version deprecations/changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- Removed deprecated argument ``obj`` in :meth:`GroupBy.get_group` (:issue:`53545`)
```

Comment:
In order to resolve a link to the documentation, can you replace the end with

    :meth:`DataFrameGroupBy.get_group` and :meth:`SeriesGroupBy.get_group`

Reply:
thank you, I corrected the note in v3.0.0.rst